themes/startwords/README.md

```diff
@@ -262,6 +262,18 @@ Either `pdf-img` or `youtube_id` should be specified.
 
 [view source](layouts/shortcodes/video.html)
 
+### source code
+
+Use to display highlighted source code from a file in a details/summary element.
+Path should be relative to content directory.
+
+Example use:
+```
+{{< source_code source_file="path/to/code.html" format="html" >}}
+```
+
+[view source](layouts/shortcodes/source_code.html)
```

Comment:
@rlskoeser Is it just me, or could the "view source" link right beneath the `source_code` example use be confusing? At first glance, I still thought the "view source" hyperlink was _part_ of the "Example use." But again, that might just be me, looking at one shortcode documentation in isolation. Someone reading the entire theme README would probably recognize that all of the shortcodes contain a "view source" link at the bottom of each section.

Reply:
Only thing I can think of is to change the "view source" hyperlink at the end of each shortcode section in the theme README to something like "view this shortcode", or "view shortcode source". Shortcode sourcecode? 🤭 Maybe this is a nonissue.

Reply:
@gwijthoff OMG it _is_ kind of confusing! I didn't even think about that, just followed the pattern from how we documented the shortcodes.  (oh no, could be "view source code shortcode source code " 🤣 ) 

I think isn't quite as bad on the [rendered version of the readme](https://github.com/Princeton-CDH/startwords/tree/chore/include-keyboards-code/themes/startwords) as it is in the markdown. I'm open to revising the label, but if we do we may want to revise the others as well.

Reply:
@rlskoeser agreed. Let's leave it as is for now. We can always revisit later.